src/components/application_manager/test/app_launch/app_launch_data_db_test.cc

```diff
@@ -99,7 +99,9 @@ class AppLaunchDataDBTest : public ::testing::Test {
     test_db_ = (res_db_->db());
 
     EXPECT_TRUE(test_db()->Open());
+#ifndef __QNX__
     EXPECT_TRUE(test_db()->IsReadWrite());
+#endif //__QNX__
```

Comment:
@BSolonenko  what is the problem here ?

Reply:
@LuxoftAKutsan there is no such method in qdb_wraper.